collection-scripts/common.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# shellcheck disable=SC2034,SC2086,SC2001,SC2068
+# be careful of 2068, oc is very sensitive with "" on variables with error error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'oc get resource/<resource_name>' instead of 'oc get resource resource/<resource_name>'
+
+export DST_DIR="must-gather"
+
+# run must-gather in the namespaces one by one
+function run_mustgather() {
+    for ns in $@; do
```

Comment:
Does not change anything here (namespaces are without spaces), but probably better "$@". It does not split arguments on spaces.